versioned_docs/version-1.0/tutorial/01-first-steps.md

```diff
@@ -15,7 +15,7 @@ Cadence introduces new features to smart contract programming that help develope
 - Resource-oriented programming, a new paradigm that pairs linear types with object capabilities to create a secure and declarative model for digital ownership
   by ensuring that resources (and their associated assets) can only exist in one location at a time, cannot be copied, and cannot be accidentally lost or deleted
 - Built-in pre-conditions and post-conditions for functions and [transactions](../language/transactions.md)
-- The utilization of capability-based security, which enforces access control by requiring that access to objects
+- The utilization of capability-based security and entitlements, which enforces access control by requiring that access to objects
```

Comment:
```suggestion
- The utilization of capability-based security and entitlements, which enforces access control by requiring that access to objects,
```

Reply:
no comma needed here. We're saying that the access is restricted, not that capability security is restricted